### ISSUE TITLE
P3 preview branch build is failing

### ISSUE THREAD

**Issue by web-padawan** [vaadin/base-starter-react#13]
The same as vaadin/base-starter-vue#13
```sh
$ yarn build
yarn run v1.9.4
$ react-scripts build
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/@polymer/polymer/lib/utils/mixin.js:13
```

Need to find out what is the problem in UglifyJS, possibly create small reproduction.

**Comment by web-padawan** [comment]
See also https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify

> even if a library provides an ES Modules version, it should still precompile other ES6 features to ES5 if it intends to support older browsers.

😐 really not sure how we could fix it ourselves.

**Comment by web-padawan** [comment]
Note this is part of #12 but there are different problems there, too.

**Comment by web-padawan** [comment]
Fixed by #16 on the `p3-preview` branch, will merge to `master` once other starters are fixed.
